package.json

```diff
@@ -39,5 +41,11 @@
   },
   "dependencies": {
     "raml2html": "^1.0.4"
-  }
+  },
```

Comment:
were inserted after testing for installing from the repository. Are not required